queries/models/dashboard_subsidio_sppo/CHANGELOG.md

```
@@ -1,5 +1,11 @@
 # Changelog - dashboard_subsidio_sppo
 
+## [8.0.3] - 2025-09-30
+
+### Alterado
+
+- Ajusta os modelos efêmeros `viagens_remuneradas_v1` e `viagens_remuneradas_v2` removendo o trecho `is incremental`.(https://github.com/prefeitura-rio/pipelines_rj_smtr/pull/)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)